### PR TITLE
Added Function sample to monitor AML pipeline run results

### LINKED ISSUE #42: Create a sample to trigger Azure ML pipelines from Azure Functions
1. Document on why and what we want to demonstrate in this sample. 
2. Create unit tests for the sample.
3. Create CI/CD pipelines for the sample.

### PR DESCRIPTION
**Pull request by aykhara**
Added new sample which demonstrate how to to monitor AML pipeline run results by using Azure Functions. 
This sample demonstrates how to retrieve such data and load into Application Insights so that you can create Dashbaord from logs.

### CODE REVIEW

File: common/pipeline_monitor/README.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    <li><a href="#repo-structure">Repo Structure</a></li>
+    <li><a href="#getting-started">Getting Started</a></li>
+    <li><a href="#analyze-run-result">Analyze run result</a></li>
+    <li><a href="#references">References</a></li>
+  </ol>
+</details>
+
+<!-- Overview -->
+# Overview
+When using Azure Machine Learning(AML) Service, there are cases when it is necessary to monitor AML pipeline run results. This sample demonstrates how to retrieve such data and load into Application Insights so that you can create dashbaord from logs.
```
Comment (h2floh) [inline common/pipeline_monitor/README.md:15]: How about explicitly mentioning that the pipeline run-duration can be measured in that way. And that additional metrics could be calculated if they are packed in the custom dimensions?

File: common/pipeline_monitor/README.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+## Query Application Insights
+
+It may take up to five minutes until the log is sent to Application Insights. 
+
+1. Open Application Insights resource.
+
+1. Select **Logs** in **Monitoring section**.
+
+1. Run Kusto query after replace variable and see the results.
```
Comment (h2floh) [inline common/pipeline_monitor/README.md]: Similar comment here, maybe state in the text that the main query result is the pipeline run duration.

**Review by h2floh — COMMENTED** [review]
Looks also great! Thanks! 👍  Please just make the main result (pipeline run duration) more clear in the README.

File: common/pipeline_monitor/README.md (hunk 1 above)
Comment (aykhara) [reply common/pipeline_monitor/README.md:15]: @h2floh Thanks for your comment! Just added explanations about the steps in this sample including run-duration and custom dimensions.

File: common/pipeline_monitor/README.md (hunk 2 above)
Comment (aykhara) [reply common/pipeline_monitor/README.md]: @h2floh Just added additional explanations here as well.

**Review by h2floh — DISMISSED** [review]

File: common/pipeline_monitor/README.md (hunk 2 above)
Comment (h2floh) [reply common/pipeline_monitor/README.md]: Thanks :)

File: common/pipeline_monitor/README.md (hunk 1 above)
Comment (h2floh) [reply common/pipeline_monitor/README.md:15]: Perfect!

**Review by h2floh — APPROVED** [review]

**Pull request merged**